msteams-platform/tabs/how-to/using-fluid-msteam.md

```diff
@@ -0,0 +1,325 @@
+author: timtwang
+ms.author: mobajemu
+description: Tutorial for integrating Fluid-powered real-time collaboration features into a Microsoft Teams tab application
+ms.localizationpriority: medium
+ms.topic: conceptual
+---
+
+# Use Fluid with Microsoft Teams
+
+This tutorial helps to integrate Fluid-powered real-time collaboration features into a [Microsoft Teams tab application](/microsoftteams/platform/tabs/what-are-tabs). By the end of this tutorial you'll be able to integrate any Fluid-powered application into Microsoft Teams and collaborate with others in real-time.
+
+In this section you'll learn the following concepts:
+
+1. How to integrate a Fluid client into a Microsoft Teams tab application.
+2. How to run and connect your Teams application to a Fluid service (Azure Fluid Relay).
+3. How to create and get Fluid Containers and pass them to a React component.
+
+For an example of how this tutorial may be used to build a more complex application, check out the [Teams Fluid Hello World](https://github.com/microsoft/FluidExamples/tree/main/teams-fluid-hello-world) example in our FluidExamples repo.
+
+## Prerequisites
+
+This tutorial requires familiarity with the following concepts and resources:
+- [Fluid Framework Overview](https://fluidframework.com/docs/)
+- [Fluid Framework QuickStart](https://fluidframework.com/docs/start/quick-start/)
+- The basics of [React](https://reactjs.org/) and [React Hooks](https://reactjs.org/docs/hooks-intro.html)
+- How to build a [Microsoft Teams Tab](/microsoftteams/platform/tabs/what-are-tabs)
+
+## Create the project
+
+1. Open a Command Prompt and navigate to the parent folder where you want to create the project, for example, `/My Microsoft Teams Projects`.
+2. Create a Teams tab application by running the below command and following [the Create a channel tab documentation](create-channel-group-tab.md#create-a-custom-channel-or-group-tab-with-nodejs).
+
+    ```cmd
+    yo teams
+    ```
+
+3. After creating the project, navigate to it with the command `cd <your project name>`.
+4. The project will use three Fluid libraries:
+
+    |Library |Description |
+    |---|---|
+    | `fluid-framework`    |Contains the IFluidContainer and other [distributed data structures](https://fluidframework.com/docs/build/dds/) that synchronize data across clients.|
+    | `@fluidframework/azure-client`   |Defines the starting schema for the [Fluid container](https://fluidframework.com/docs/build/containers/).|
+    | `@fluidframework/test-client-utils` |Defines the `InsecureTokenProvider` needed to create the connection to a Fluid service.|
+
+    Run the following command to install the libraries.
+
+    ```cmd
+    npm install @fluidframework/azure-client fluid-framework @fluidframework/test-client-utils
+    ```
+
+## Code the project
+
+1. Open the file `/src/client/<your tab name>` in your code editor. Create a new file named `Util.ts`. Then add the following import statements:
```

Comment:
1. Open the file `/src/client/<your tab name>` in your code editor. 
2. Create a new file as `Util.ts` and add the following import statements: